libGeoIP/GeoIP.c

```diff
@@ -1615,7 +1615,7 @@ const char *GeoIP_country_code_by_name_v6_gl(GeoIP * gi, const char *name,
 {
     int country_id;
     country_id = GeoIP_id_by_name_v6_gl(gi, name, gl);
-    return (country_id > 0) ? GeoIP_country_code[country_id] : NULL;
+    return (country_id > 0) ? GeoIP_code_by_id(country_id) : NULL;
```

Comment:
Yeah `0` is a very special number here. I tried it the sane way originally but that caused test failures.